README.md

```diff
@@ -81,9 +81,14 @@ This is skipped if the requested file already has an extension.
 
 ##### index
 
-By default send supports "index.html" files, to disable this
-set `false` or to supply a new index pass a string or an array
-in preferred order.
+By default *send* supports "index.html" files. To disable this, set index option
+to `false`, or you can be able to supply a new index passing a string with the
```

Comment:
"you can be able to" doesn't sound right. I think probably just remove those words?